### PR TITLE
chore(lib): bump milli to 0.25.0

### PR DESCRIPTION
**Pull request by MarinPostma**
bump milli

### CODE REVIEW

File: meilisearch-lib/src/error.rs [hunk 1]
```diff
@@ -41,6 +41,7 @@ impl ErrorCode for MilliError<'_> {
                     UserError::CriterionError(_) => Code::InvalidRankingRule,
                     UserError::InvalidGeoField { .. } => Code::InvalidGeoField,
                     UserError::SortError(_) => Code::Sort,
+                    UserError::InvalidMinTypoWordLenSetting(_, _) => unreachable!(),
```
Comment (ManyTheFish) [inline meilisearch-lib/src/error.rs:44]: Not sure but, it should be better to return at least an internal error, no? 🤔

Comment (MarinPostma) [reply meilisearch-lib/src/error.rs:44]: this error cannot be reached now, but will in two PR :) (#2258)

Comment (ManyTheFish) [reply meilisearch-lib/src/error.rs:44]: Ah ok! Thanks!

**Comment by MarinPostma** [comment]
bors merge

**Pull request merged**